os/arch/arm/src/amebasmart/amebasmart_timerisr.c

```diff
@@ -94,6 +91,13 @@ int up_timerisr(int irq, uint32_t *regs)
 	  delta_ticks = 1;
 #endif
 
+    u32 ticks_to_process = delta_ticks;
+    while (ticks_to_process > 0) {
+      /* process missing ticks */
+      sched_process_timer();
+      ticks_to_process--;
+    }
+
```

Comment:
I think, this code can make duplicated compensate missing tick.
https://github.com/Samsung/TizenRT/blob/d4a807c961546331a8957fd4c1e9aa364808026b/os/pm/pm_wakehandler.c#L64-L77

pm_wakehandler is conpensating missing tick.
but this code looks conpensating tick irq loss.

Reply:
```
void up_pm_board_sleep(void (*wakeuphandler)(clock_t, pm_wakeup_reason_code_t))
{
...

	/* Arch timer is running at 50Mhz */
	arm_arch_timer_set_compare(arm_arch_timer_count() + 50000);
	arm_arch_timer_int_mask(0);
}
```
the timer is setted latest count in above code, so duplicating compensation is not ocurred right?

well, then why did we need below code for CONFIG_PM?
```
#ifdef CONFIG_PM
	  if (arm_arch_timer_count() < last_cycle) {
		  return -1;
	  } else {
		  delta_ticks = (uint32_t)((arm_arch_timer_count() - last_cycle) / SYSTICK_RELOAD) + 1;
	  }
#else
```